synapse/config/saml2_config.py

```diff
@@ -226,6 +234,11 @@ def generate_config_section(self, config_dir_path, server_name, **kwargs):
           #
           #mxid_mapping: dotreplace
 
+          # A third-party module can be provided here as a custom solution to mapping the
```

Comment:
Bit debatable if it's third-party: who are the first and second parties?

```suggestion
          # An external module can be provided here as a custom solution to mapping the
```